### ISSUE TITLE
Narrator/NVDA doesn't read an error message when it is set to an ErrorProvider 

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#2390]
* .NET Core Version: 5.0.100-alpha1-015748
* Have you experienced this same bug with .NET Framework?: Yes

**Original bug:**
- [Bug 98789](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/987891)

**Problem description:**
- If you set an error message for a control using ErrorProvider.SetError method ScreenReader won't announce an error text

**Actual behavior:** 
- ScreenReader is silent

**Expected behavior:**
- ScreenReader announces an error text

**Minimal repro:**
- Create .Net 5.0 app
- Add some control and ErrorProvider to the app
- Use ErrorProvider.SetError method to set an error to the control
- Run and listen to ScreenReader

**Comment by vladimir-krestov** [comment]
It seems it is an OS regression bug, so I closed this Issue. I will reopen it if necessary.

**Comment by John-Qiao** [comment]
@vladimir-krestov  I still can reproduce this issue using the latest .NET Core 5.0 SDK on Win10 1909(OS Build 18363.720), please confirm it.
![ErrorProvider](https://user-images.githubusercontent.com/45864985/78973525-b3a1ba00-7b42-11ea-8b43-8a73b632521e.png)

**Comment by vladimir-krestov** [comment]
@John-Qiao, there are 2 ways how to announce an `ErrorProvider` ToolTip text:
- **Using a mouse**: mouse over an `ErrorProvider` red cross icon, you will see a ToolTip with a text. If some user can't read the text he/she has to use `"Narrator+0"` keyboard combination (`"Narrator"` key is settable key, it is `CapsLock` by default, `"0"` is the keyboard number button on the top of your keyboard). This combination makes Narrator announce a last shown ToolTip.
- **Using a keyboard**: some users can't use a mouse. In this case, they can use `"Narrator+keyboard arrows"` (e.g. `CapsLock+Right`) to move through form controls in the scan mode. When Narrator focuses on an `ErrorProvider` red cross icon it announces an `ErrorProvider` ToolTip text.

@John-Qiao, could you please check if it is true for different OS versions (e.g. 20H2 or 19H(1809)) and it works well as I said?

Actually, I think they are not comfortable ways. I would prefer to see that Narrator announces an `ErrorProvider` ToolTip text when its icon appears (when we call `SetError` method). The fix for this is ready - PR  #2391. 

@merriemcgaw, how do you think, will we make Narrator to announce the `ErrorProvider` text when its icon appears? Or just use the existing ways?

**Comment by John-Qiao** [comment]
@vladimir-krestov I have tested this scenario on 3 OS, please check the below test results:

In Windows version 1809 OS, the _ErrorProvider_ text can be announced by Narrator automatically when move mouse on the error icon.
![OS-1809-TestResult](https://user-images.githubusercontent.com/45864985/98910248-e3140d00-24fd-11eb-89e8-89e2de35c32f.gif)

In Windows version 2004 OS and 20H2 OS, the _ErrorProvider_ text cannot be announced by Narrator automatically when move mouse on the error icon. That only works when **using a mouse** or **using a keyboard** as you said.
![CapsLock+0](https://user-images.githubusercontent.com/45864985/98910284-ead3b180-24fd-11eb-84b8-e9f6eb87bce3.png)
![CapsLock+Right](https://user-images.githubusercontent.com/45864985/98910292-ee673880-24fd-11eb-9d01-c15ea8e3e73e.png)

Either targeting to .NET 5.0 or the latest .NET 6.0 both have the same above test results in different OS.

**Comment by vladimir-krestov** [comment]
@John-Qiao, thank you so much for testing!
Yes, auto Narrator announcing of a ToolTip text worked in 1809 OS version and before. For the next versions a user have to use "Narrator+0" combination to announce a ToolTip text. It is expected for later versions of Narrator by design. In the rest, yes, we have 2 ways how to announce an ErrorProvider text.
@merriemcgaw, is it enough from our side? Should we announce an ErrorProvider text when its red icon appears?

**Comment by vladimir-krestov** [comment]
Described 2 ways are recognized uncomfortable for users. This issue should be reworked.
